Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -46,6 +46,8 @@ protected class MemoryCacheItem
 
         private static readonly SemaphoreSlim _cacheFolderSemaphore = new SemaphoreSlim(1);
         private static readonly Dictionary<string, Task<BitmapImage>> _concurrentTasks = new Dictionary<string, Task<BitmapImage>>();
+        private static readonly Dictionary<string, Task> _concurrentPrecacheTasks = new Dictionary<string, Task>();
```

Comment:
Heaving two collections for concurrent tasks seems messy, but type of task is different.